src/hooks/useDropdownOptions.ts

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export interface Item {
+  [key: string]: string | number;
+}
+
+/**
+ * Transforms your array of objects into the shape our form system expects.
+ * Our form system expects an Array of Objects with a shape of [{label: String, value: String}]
+ * @param {string} label
+ * @param {string} value
+ * @param {array} items [{item}]
+ */
+export function useDropdownOptions(
+  label: string,
+  value: string,
+  items?: Item[]
+): {label: string | number; value: string | number | null}[] {
```

Comment:
Inline types can look pretty noisy. Thoughts on defining the type, assigning the type to a variable then using the type variable vs an inline type?

Reply:
For sure - but now that I'm looking at it again, the inferred return type is the exact same thing, so I think I could forgo specifying a return type altogether